### PR TITLE
Add more getting started infos in README.md

### PR DESCRIPTION
**Pull request by KikiManjaro**
Feel free to add modifications to what i've written :) you know the project better than me

### CODE REVIEW

**Review by flomonster — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -22,14 +22,28 @@ Please don't build any serious projects with OSRD unless you are prepared to be
...
-You'll need:
- - Docker
- - Docker Compose
+### What you'll need
 
+- [Docker Compose](https://github.com/docker/compose)
+
+You can find install process right [here](https://docs.docker.com/compose/install/).
+
+Run this command in root of this project should start the docker :
```
Comment (multun) [inline README.md]: This command shouldn't run as root, you only have to be in the proper group. Typography rules in English also are different: there are no spaces before colons.

File: README.md [hunk 2]
```diff
@@ -22,14 +22,28 @@ Please don't build any serious projects with OSRD unless you are prepared to be
...
 ## Getting Started
 
-You'll need:
- - Docker
- - Docker Compose
+### What you'll need
 
+- [Docker Compose](https://github.com/docker/compose)
+
+You can find install process right [here](https://docs.docker.com/compose/install/).
```
Comment (multun) [inline README.md]: Can you keep the list just like previously, and just add the links to the install page? People also still need to install docker, installing just docker-compose wont work (even though both are packaged together on windows)

**Review by multun — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (KikiManjaro) [reply README.md]: Ok, done :)

File: README.md (hunk 1 above)
Comment (KikiManjaro) [reply README.md]: As I'm saying :
This command should be run in root of this project (such as 'osrd/')
I'm not saying that this command should be run as root user. Please tell me if you have a better way to explain that.

I've removed the space, my bad :)

**Comment by multun** [comment]
@KikiManjaro just so you know, we have not forgotten this PR, we'd just like to add some more stuff to it

**Comment by multun** [comment]
We're making a documentation website which already contains most of what's in there. I'll change this PR to add a link to this documentation once it's in a satisfying state

https://dgexsolutions.github.io/osrd-website/fr/developers/

**Comment by multun** [comment]
Time has passed, and a lot of things have changed (including the README). I don't think it makes sense to merge anymore

**Pull request closed without merging**